docs/tutorial/typesfuns.rst

```diff
@@ -860,12 +860,12 @@ Dependent Pairs
 
 Dependent pairs allow the type of the second element of a pair to depend
 on the value of the first element. Traditionally, these are referred to
-as “sigma types”:
+as “sigma types”, so in older code you might see Sigma rather than DPair:
```

Comment:
Sigma is deprecated, but I see your point. Maybe this could be rephrased so that that is more apparent (that it is deprecated now).